apps/backend/src/modules/groups/group.controller.ts

```diff
@@ -115,6 +124,36 @@ class GroupController extends BaseController {
 			status: HTTPCode.CREATED,
 		};
 	}
+
+	/**
+	 * @swagger
+	 * /groups/{name}:
+	 *   delete:
+	 *     description: Delete a group by its name
+	 *     parameters:
+	 *       - in: path
+	 *         name: name
+	 *         required: true
+	 *         schema:
+	 *           type: string
+	 *     responses:
+	 *       204:
+	 *         description: Group deleted successfully
+	 *       404:
+	 *         description: Group not found
+	 */
+
+	private async deleteGroup(
+		options: APIHandlerOptions<{ params: { id: number } }>,
+	): Promise<APIHandlerResponse> {
+		const { id } = options.params;
+		await this.groupService.delete(Number(id));
```

Comment:
Since `id` is already typed as a number, there's no need to convert it to a Number.